grpc-server-security-jwt/build.gradle

```diff
@@ -0,0 +1,55 @@
+
+plugins {
+    id "net.ltgt.apt" version "0.21"
+    id 'com.google.protobuf' version '0.8.15'
+}
+
+dependencies {
+
+    annotationProcessor "io.micronaut:micronaut-inject-java:$micronautVersion"
+    testAnnotationProcessor "io.micronaut.security:micronaut-security-annotations:$micronautSecurityVersion"
```

Comment:
Can't seem to get this to work with `@Secured` in my integration test.

Reply:
I think I realize why.  Looks like this approach may not support the `@Secured` annotation.

Reply:
I can use a similar approach to the `intercept-url-pattern-map`.  Should I do go down that route?

Reply:
I've implemented a `intercept-method-patterns` approach for role based security.